website/docs/docs/cloud/about-cloud/regions-ip-addresses.md

```diff
@@ -11,10 +11,17 @@ dbt Cloud is [hosted](/docs/cloud/about-cloud/architecture) in multiple regions
 
 | Region | Location | Access URL | IP addresses | Developer plan | Team plan | Enterprise plan |
 |--------|----------|------------|--------------|----------------|-----------|-----------------|
-| North America [^1] | AWS us-east-1 (N. Virginia) | cloud.getdbt.com | 52.45.144.63 <br /> 54.81.134.249 <br />52.22.161.231 | ✅ | ✅ | ✅ |
+| North America multi-tenant [^1] | AWS us-east-1 (N. Virginia) | cloud.getdbt.com | 52.45.144.63 <br /> 54.81.134.249 <br />52.22.161.231 | ✅ | ✅ | ✅ |
+| North America Cell 1 [^1]       | AWS use-east-1 (N.Virginia) | {account prefix}.us1.dbt.com | [Located in Account Settings](#locating-your-dbt-cloud-ip-addresses) | ❌ | ❌ | ❌ |
```

Comment:
this is cool! one thing i'd suggest is possibly having a separate section or page dedicated to explaining access URLs in more detail since this table is getting more sophisticated.[snowflake](https://docs.snowflake.com/en/user-guide/organizations-connect) has a semi-related page that explains URLs in more detail.

also maybe a bit more context on what account prefix means since this is kind of new?

Reply:
+1 -- @mirnawong1 I think it's a good idea to include a dedicated page explaining access URLs